packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -57,54 +64,92 @@ export const ArticlePostCard = forwardRef(function PostCard(
   return (
     <FeedItemContainer
       {...props}
-      className={getPostClassNames(post, className, 'min-h-[22.5rem]')}
+      className={getPostClassNames(
+        post,
+        classNames(
+          className,
+          showFeedback && '!p-0',
+          !showFeedback && hasBeenRead && styles.read,
```

Comment:
This should not be needed. it would be overwritten from `getPostClassNames` already.